widget/embedded/src/containers/Inputs/Inputs.tsx

```diff
@@ -55,20 +60,21 @@ export function Inputs(props: PropTypes) {
 
   const priceImpactInputCanNotBeComputed = !canComputePriceImpact(
     selectedQuote,
-    inputAmount,
+    inputUsdAmount.toString(),
     inputUsdValue
   );
 
   const priceImpactOutputCanNotBeComputed = !canComputePriceImpact(
     selectedQuote,
-    inputAmount,
+    inputUsdAmount.toString(),
```

Comment:
It should be like the input here:
the second argument of the `canComputePriceImpact` function is the input amount, not the input USD amount.

I think this bug already existed in production because we passed the input amount instead of the output amount.

Reply:
fixed